src/main/resources/assets/pipez/lang/de_de.json

```diff
@@ -0,0 +1,85 @@
+{
+  "itemGroup.pipez": "Röhrez",
+  "block.pipez.item_pipe": "Item Röhre",
```

Comment:
"Röhre" doesn't fit very well. I would suggest something like "Rohr" or "Leitung"

Reply:
will change to Rohr